shell.nix

```diff
@@ -9,29 +9,11 @@ in { pkgs ? import (_pkgs.fetchFromGitHub {
 
 with pkgs;
 
-let
```

Comment:
How does this work with the CI now?

Reply:
Dont know if I understand the questions, sorry. Currently, the `shell.nix` file is only needed for the `ci-checks.sh` script.

Reply:
That answers the question 👍🏻 ; I'd forgotten.